### ISSUE TITLE
Module mdx_outline not found

### ISSUE THREAD

**Issue by henrik-garbergs** [doorstop-dev/doorstop#470]
Does anyone recognize this? When I run the Doorstop tests, i.e. "make ci" in the Doorstop repo, it can apparently publish without errors, all tests are passed. But when I run the _doorstop publish_ command from the terminal it fails:

```
> doorstop publish all ./html/
building tree...
loading documents...
publishing tree to '/home/petloo/repos/src/vision/requirements/html'...
Traceback (most recent call last):
File "markdown/__init__.py", line 231, in build_extension
File "importlib/__init__.py", line 126, in import_module
File "<frozen importlib._bootstrap>", line 994, in _gcd_import
File "<frozen importlib._bootstrap>", line 971, in _find_and_load
File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'mdx_outline'
```

**Comment by henrik-garbergs** [comment]
Apparently I can at least access mdx_outline from the Python command interpreter:

```
Python 3.6.9 (default, Nov  7 2019, 10:44:02) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import markdown
>>> src = "test"
>>> html = markdown.markdown(src, ['mdx_outline'])
>>> print(html)
<p>test</p>
```

**Comment by jacebrowning** [comment]
How did you install Doorstop? I think we should probably recommend `pipx` for global installations: https://pipxproject.github.io/pipx/

**Comment by henrik-garbergs** [comment]
I got this error from a Doorstop instance that I built myself, i.e. "make exe" from the cloned Doorstop repo.

**Comment by henrik-garbergs** [comment]
It works fine if I do "make dist" and then use `pip` to install the built wheel file. 
The problem with "make exe" seem to be caused by an incorrect setting in sys.path. I will continue to investigate, but for now I can live with using "make dist".

**Comment by henrik-garbergs** [comment]
I found the problem. Pyinstaller doesn't detect when the mdx_outline module is imported since it's imported as an _extension_ through the markdown module. So I added this in the makefile to explicitly point out that module. There were some additional files missing as well, which I added. Here is the pull request:
https://github.com/doorstop-dev/doorstop/pull/472

By the way, github says that my fork (my own dev branch) is behind the doorstop-dev:develop branch by 10 commits. Do I need to rebase my own branch? I don't know how to do that.
